jsonutils/fixtures_test/harness.go

```diff
@@ -13,8 +13,8 @@ import (
 	"regexp"
 	"testing"
 
-	"github.com/stretchr/testify/assert"
-	"github.com/stretchr/testify/require"
+	"github.com/go-openapi/testify/v2/assert"
+	"github.com/go-openapi/testify/v2/require"
 	yaml "go.yaml.in/yaml/v3"
```

Comment:
I was surprised to see a yaml import.

I thought I would see a enable/yaml one

Then I looked why you needed yaml import

And I feel like, you can now simplify

https://github.com/fredbi/swag/blob/e8de8a1b47e96efe146006b65f70f0dc2d96591d/jsonutils/fixtures_test/harness.go#L296